### PR TITLE
Add stamina penalty to gas masks with no internals.

### PR DESCRIPTION
**Pull request by TDHooligan**
[BALANCE] [OBJECTS]
## About the PR 

Wearing a clothing/mask/gas with no internals applies a -2 stamina regen penalty. 
This does not affect:
 - Slasher mask
 - Nukeops masks
 - Nukie / NT Special Ops masks 
 - Cursed clown masks
 - Voice mask

## Why's this needed? 
Require at least some attention to wearing gas masks 24/7, rather than being a one-and-done item.
people have been calling for gas masks to be a little more interactive for years.

## Changelog 

```changelog
(u)TDHooligan
(*)Gas masks are harder to breathe through if internals aren't on.
```

### CODE REVIEW

**Comment by flappybatpal** [comment]
Seems like a reasonable solution.

**Comment by Valtsu0** [comment]
Imo this falls into the nerfs antags wearing gas masks, doesn't really affect crew pitfall

**Comment by TDHooligan** [comment]
>Imo this falls into the nerfs antags wearing gas masks, doesn't really affect crew pitfall. Also makes very little sense from a realism perspective.

it nerfs anyone wearing gas masks constantly. antag specific gas masks are also exempt. 

i don't see how it is an 'antag wearing gas mask' nerf besides the fact that antags are traditionally the targets of gas weaponry. whereby it's an issue that gas weapons are useless for antags (which is more to do with how poison smoke is shit)

\> realism
but if you must have 'realism' (which isnt a good goal as long as you're able to retain suspension of disbelief). 
breathing through a multilayer, microporous filter is not easy. breathing from a pressurized air hookup is.

**Comment by Sbmhawk** [comment]
> > Imo this falls into the nerfs antags wearing gas masks, doesn't really affect crew pitfall. Also makes very little sense from a realism perspective.
> 
> it nerfs anyone wearing gas masks constantly. antag specific gas masks are also exempt.
> 
> i don't see how it is an 'antag wearing gas mask nerf'

Gas masks are very commonly used by antags to hide identity, which is less needed for crew. If it's not in the pitfall, it's teetering on that edge

**Comment by AXLFAN2000** [comment]
I don't feel like it's a hard antag nerf in the same way some of the other proposed changes have been, because it's counterable without removing the mask.  I like the inclusion of the internals negating it

File: code/obj/item/clothing/masks.dm [hunk 1]
```diff
@@ -111,13 +111,39 @@
...
 		setProperty("disorient_resist_eye", 10)
 
+	equipped(mob/user, slot)
+		. = ..()
+		if (internals_penalty && ishuman(user))
+			processing_items.Add(src)
+
+	unequipped(mob/user)
+		. = ..()
+		if (internals_penalty)
```
[the remark this answers is not in the document]
Comment (TDHooligan) [reply code/obj/item/clothing/masks.dm:132]: covering niche cases like mob getting deleted or being null somehow. possibly being overprotective where i don't need to be tho

**Comment by colossusqw** [comment]
This is better than both my PRs, my gratitude upon thee.

**Comment by 444Portal** [comment]
Out of all of these, I like this the most. Dedicate a pocket slot or a belt slot to ignore the penalty, seems good to me.

If how visible the effect is a concern, maybe make a status effect appear?

**Comment by Sbmhawk** [comment]
A combination of being able to cut off the filter, and a status effect would solve both my problems with it personally. You would still be able to use it as fashion/concealment without getting hit by it, and it wouldn't br a silent noob trap.

**Comment by Retrino** [comment]
i agree that this does hit antags more than anything. gas masks are super common (at least on rp) to combo with a voice changer for antag gimmicks. your average crew member probably wont feel this nerf much, since as others have mentioned stamina is most prevalent in combat. that said i prefer this change over the other proposals, but i wish listening post got extended tanks like nukies/salvs do

**Comment by Demigurian** [comment]
> I just.. don't see the need to "balance" gas masks of all things?
Kinda resonating with this to be honest. I know they are pretty robust for something so easily available but are they affecting the game loop that severely to need active balancing??
And as far as interaction goes, its only affecting (passively) a mechanic which is needed mostly for combat purposes (again, normal traitors taking the L).
I mean, I don't mind the doc's working in the miasmabay, a few tiders just for the sake of general paranoia or actual antags having a robust item for free.

**Comment by TDHooligan** [comment]
> I just.. don't see the need to "balance" gas masks of all things?

atm some disincentive to everyone wearing them constantly is all the issue is. 
Personally i'd rather every slot have at least some competition, given that gas masks are fire and forget best-in-slot items unless you get hold of something extravagantly rare. Internals cover the already minimal value of medical face masks.

> sunglasses are also like this

sunglasses compete with a series of useful bits of headwear outside of robusting or robotics. see sec NVG gear, Health HUDs, science HUDs, on top of that, they're scarce.

the problem with gas masks becoming scarce is that: things that murder gas mask wearers will always target huge swathes of people (smoke bombs), while the things that affect sunglasses wearers are smaller in scope (flashes)

**Comment by flrsh** [comment]
I agree with Leah and Earthfire. If we want people to not be wearing masks all the time, I think that extends past just gas masks, and we should look into a (controversial) make-people-take-off-their-masks-to-consume-things-mechanic or something.

**Comment by TDHooligan** [comment]
it's a feature on /tg/ but all it means is you click off a gas mask for half a second to eat/swallow whatever and then slap it on. still leaves the problem of there being no reason not to wear gas masks for an entire shift.

**Comment by PastaSpaghett** [comment]
Maybe don't add this one to Nukie gas masks, that would be a much bigger nerf to Nukies than anyone else.

**Comment by PastaSpaghett** [comment]
Maybe don't give the SWAT masks this, it would be nice to give those something to elevate them above regular gas masks.

**Pull request closed without merging**